front/pages/api/auth/[auth0].ts

```diff
@@ -66,9 +65,6 @@ const afterCallback: AfterCallbackPageRoute = async (
 
     const params = new URLSearchParams();
 
-    // Add the silent auth params.
-    params.set("prompt", "none");
```

Comment:
When switching region we should not hide the prompt if needed (as apps are differents it may require consent)